src/layouts/AssistenzaPage.jsx

```diff
@@ -0,0 +1,839 @@
+                    </label>
+                    <span className={classes.maxLengthLabel}>
+                      Massimo <span className="max-length-number">{textareaMaxLength}</span> caratteri
+                    </span>
+                  </div>
+                </Col>
+              </Row>
+              <input type="hidden" id="external" name="external" value="1" />
+              <input type="hidden" name="origin" value="Area pubblica" />
+              <input type="hidden" name="recordType" value="0127Y00000229Rc" />
+              <Row className="mt-5">
+                <Col xs={12}>
+                  <ReCAPTCHA
+                    sitekey="6LfW56weAAAAAIWHJnwlQ2lHNRCcd04QLYQyamww"
+                    onChange={onChangeCaptcha}
+                    className={`${formFilled ? '' : 'd-none'}`}
+                    id="captcha-container"
+                  />
+                </Col>
+              </Row>
+              <div className="justify-content-center flex-column align-items-start justify-content-md-start px-0 py-0 mt-5">
+                <p className={classes.modalFooterLabel}>
+                  Cliccando su INVIA dichiaro di aver letto e compreso{' '}
+                  <a target="_blank" href={privacy.linkTo} rel="noreferrer">
+                    l'informativa privacy
+                  </a>
+                </p>
+                <div className="d-flex">
+                  <input type="submit" name="submit" disabled value={sendButtonLabel} className="btn btn-primary" />
+                </div>
+              </div>
+            </form>
+            <div className={classes.notification} role="alert" aria-labelledby="not2dms-title" id="not2dms2">
+              <h5 id="not2dms-title2">
+                notifiche
+                <svg className="icon" role="img" aria-label=""></svg>
+              </h5>
+              <p></p>
+              <button
+                type="button"
+                className="btn notification-close"
+                aria-label="Chiudi"
+                aria-describedby="not2dms-title"
+              >
+                <svg
+                  width="19"
+                  height="19"
+                  viewBox="0 0 19 19"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                  focusable="false"
+                  role="img"
+                  aria-label="Chiudi"
+                >
+                  <rect
+                    x="17.3242"
+                    y="0.5"
+                    width="1.49987"
+                    height="24.4978"
+                    transform="rotate(45 17.3242 0.5)"
+                    fill="#5C6F82"
+                  />
+                  <rect y="1.56055" width="1.49987" height="24.4978" transform="rotate(-45 0 1.56055)" fill="#5C6F82" />
+                </svg>
+              </button>
+            </div>
+          </Col>
+        </Row>
+        <script src="https://www.google.com/recaptcha/api.js"></script>
```

Comment:
we shouldn't need this as we have the library for that.

Reply:
I removed it